app/src/ui/autocompletion/emoji-autocompletion-provider.tsx

```diff
@@ -63,7 +75,11 @@ export class EmojiAutocompletionProvider
     for (const emoji of this.allEmoji.keys()) {
       const index = emoji.indexOf(needle)
       if (index !== -1) {
-        results.push({ emoji, matchStart: index, matchLength: needle.length })
+        results.push({
+          title: emoji,
+          matchStart: index,
+          matchLength: needle.length,
+        })
       }
     }
```

Comment:
Unless I missing something, with that approach, when you write let's say `:tad` and select 🎉 , the unicode character doesn't get inserted. However with this suggestion it does:

```suggestion
    for (const [key, emoji] of this.allEmoji) {
      const index = key.indexOf(needle)
      if (index !== -1) {
        results.push({
          title: key,
          emoji: emoji?.emoji,
          matchStart: index,
          matchLength: needle.length,
        })
      }
    }
```

Does that make sense?

Reply:
It sure does!